README.md

```diff
@@ -47,20 +56,85 @@ and ask!
 
 ## When to Create a New Enhancement
 
+Enhancements should be related to work to be implemented in the near
+future. If you have an idea, but aren't planning to implement it right
+away, the conversation should start somewhere else like the aos-devel
+mailing list.
+
 Create an enhancement here once you:
 
 - have circulated your idea to see if there is interest
 - (optionally) have done a prototype in your own fork
 - have identified people who agree to work on and maintain the enhancement
   - many enhancements will take several releases to complete
 
-## Why are Enhancements Tracked
-
-As the project evolves, its important that the OKD community understands how we
-build, test, and document our work.  Individually it is hard to understand how
-all parts of the system interact, but as a community we can lean on each other
-to build the right design and approach before getting too deep into an
-implementation.
+## How are Enhancements Reviewed and Approved?
+
+The author of an enhancement is responsible for managing it through
+the review process, including soliciting feedback on the pull request
+and in meetings, if necessary.
+
+Each enhancement should have at least one "approver" and several
+reviewers designated in the header of the document.
+
+The approver assists authors who may not be familiar with the process,
+the project, or the maintainers. They may provide advice about who
+should review a specific proposal and point out deadlines or other
+time-based criteria for completing work. The approver is responsible
+for recognizing when consensus has been reached so that a proposal is
+ready to be approved, or formally rejected. In cases where consensus
+is not emerging on its own, the approver may also step in as a
+mediator. The approver do not need to be a subject-matter expert for
+the subject of the design, although it can help if they are.
+
+Choosing the appropriate approver depends on the scope of an
+enhancement. If it is limited in scope to a given team or component,
+then a peer or lead on that team or pillar is appropriate.  If an
+enhancement captures something more broad in scope, then a member of
+the OpenShift staff engineers team or someone they delegate would be
+appropriate.  Examples of broad scope are proposals that change the
+definition of OpenShift in some way, add a new required dependency, or
+change the way customers are supported.  Use your best judgement to
+determine the level of approval needed.  If you’re not sure, ask a
+staff engineer to help find a good approver by posting in
+`#forum-arch` on the CoreOS Slack server and tagging
+`@aos-staff-engineers`.
```

Comment:
as w/ the aos-devel suggestion above, it bothers me a little that this is basically an internal-only process, but it accurately reflects the process we have/follow today.  So i guess rather than blocking this on "we need to define the community process before we document our EP workflow", i guess i'll just make a note that this feels like a gap in our EP workflow.

Reply:
Agreed